safe/processors/post_processors.py

```diff
@@ -283,3 +283,21 @@
         }
     }
 }
+
+# post_processor_filter_example = {
```

Comment:
Is this will be used in unit test?

Reply:
nope, just for documenting usage. in the tests I mock